launch.sh

```diff
@@ -196,8 +200,15 @@ fail_if_root
 
 print_delimiter
 
-# Get the absolute path of the directory where this script is located
-SRC_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+# Check if any arguments were supplied and launch wizard if not
+WIZARD_FILE_DIR="${SCRIPT_DIR}/wizard.sh"
+if [[ $# -eq 0 ]]; then
+    if [[ ! -f $WIZARD_FILE_DIR ]]; then
+        print_red "Canno't locate ${WIZARD_FILE_DIR}\nPlease make sure all script files are present."
+        exit 1
```

Comment:
```suggestion
        print_red "Cannot locate ${WIZARD_FILE_DIR}.\nPlease make sure all script files are present."
```
nit: print_red already exits with 1, so this isn't needed.